src/routes/templates/templates.json

```
@@ -722,6 +722,16 @@
 		"tags": ["blog", "code splitting", "lazy loading", "seo", "ssr", "templates", "typescript"],
 		"stars": 0
 	},
+	{
+		"title": "svelte-ts-tailwind-template",
+		"url": "https://github.com/colinbate/svelte-ts-tailwind-template",
+		"description": "Svelte app template with TypeScript and TailwindCSS.",
+		"npm": "",
+		"addedOn": "2022-02-28",
+		"category": "Svelte",
+		"tags": ["templates", "typescript"],
+		"stars": 0
+	}
```

There is a missing comma here
```suggestion
	},
```